numpy/_core/tests/test_stringdtype.py

```diff
@@ -551,11 +551,11 @@ def test_concatenate(string_list):
 
 def test_resize_method(string_list):
     sarr = np.array(string_list, dtype="T")
-    if IS_PYPY:
+    try:
```

Comment:
Same here

Reply:
This is the failed test for this test case- 

![Screenshot 2024-12-20 at 2 22 10 AM](https://github.com/user-attachments/assets/3b33d326-fddf-4a14-872c-5e3cbb303b37)